### DISCUSSION TITLE
linux ?

### DISCUSSION THREAD

**Discussion by Pewful2021** [seerge/g-helper#2778] [General]
so i just use g helper for fan control but now that i switched to linux (pop os) i cant control my fans and my laptop is overheating.
Any ideas ?
Laptop : FX516PE

**Comment by seerge** [comment]
@Pewful2021 you can use linux alternatives https://asus-linux.org/

**Comment by Pewful2021** [comment]
how do i install it im new to linux

**Reply by seerge** [reply]
@Pewful2021 just check documentation on the website i linked :) it has manuals and also link to the discord channel for support.

**Comment by nilaysriv** [comment]
ASUS Linux does work but on TUF laptops, it doesn't have controls like undervolting, setting up power limits, gpu modes, refresh rates and so many options. G-Helper is the only thing keeping me  (and pretty sure a lot of users) from switching to Linux full time. I wish you guys would collab and make a powerful tool for linux users like you made G-Helper for windows users :)

**Reply by seerge** [reply]
@nilaysriv we actually do collab in a way. 

I think Luke (author of asusctl) was using some of the reverse-engineering findings that I had. And I in turn used some of his findings for RGB controls when I was starting this app :)

**Reply by nilaysriv** [reply]
That's awesome. I hope Linux gets better AMD APU tuning and something a bit closer to G-Helper. It'd be amazing

**Comment by pmkelly545** [comment]
I came here to second g-helper on Linux. I think your UI is simple and clean. It would be awesome to use it on Bazzite.

**Reply by seerge** [reply]
@pmkelly545 what exact features from G-Helper do you miss at **asusctl** app on linux ?

**Reply by AlexRD-42** [reply]
> @pmkelly545 what exact features from G-Helper do you miss at **asusctl** app on linux ?

Probably setting TDP values for GPU and CPU, with the different profiles. I also am not switching from windows because of your app haha

**Reply by iLukiji** [reply]
Same for me as well. Asus linux just doesn't work well on my new laptop. I also can't use just my NVIDIA card.

**Reply by utajum** [reply]
Maybe this will work for you guys https://g-helper-linux.elevatech.xyz/

If something does not work, open an issue with details

**Reply by i2** [reply]
> Maybe this will work for you guys https://g-helper-linux.elevatech.xyz/

Have you thought about a merge request of your project to the G-Helper (if possible)?

**Reply by utajum** [reply]
That is not really possible unfortunate since the platforms are way too different, but it is an interesting idea :)
